qemu/tests/cfg/hotplug_mem_repeat.cfg

```diff
@@ -11,6 +11,12 @@
     mem_fixed = 4096
     size_mem = 128M
     maxmem_mem = 40G
+    Linux:
+        mem_unplug_test = yes
+        RHEL.7:
+            mem_unplug_test = no
+    Windows:
+        mem_unplug_test = no
```

Comment:
How about remove L18~19 as it is the default value ?

Reply:
Done!